config/secrets/custom-smtp-secret.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: quota-secret
+objects:
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: addon-managed-api-service-parameters
```

Comment:
We have another dummy secret in this folder which we're using for QUOTA configuration. I think we should rename the yaml file to `addon-managed-api-sevices-paramaters` as it covers all ocm parameters to the installation. wdyt?

Reply:
@Boomatang Can you create a JIRA so to follow up on this

Reply:
Related Jira: https://issues.redhat.com/browse/MGDAPI-3962